Funcky.Test/Extensions/EnumerableExtensions/InspectTest.cs

```diff
@@ -1,12 +1,23 @@
 using System.Collections.Generic;
 using Funcky.Extensions;
+using Funcky.Test.TestUtils;
 using Xunit;
 using static Funcky.Functional;
 
 namespace Funcky.Test.Extensions.EnumerableExtensions
 {
     public sealed class InspectTest
     {
+        [Fact]
+        public void InspectIsEnumeratedLazily()
+        {
+            var sideEffect = 0;
+            var doNotEnumerate = new FailOnEnumerateSequence<object>();
+
+            _ = doNotEnumerate.Inspect(n => { ++sideEffect; });
```

Comment:
Why aren't you using `FailOnCall.Function` here?

Reply:
Or `NoOperation` since laziness is already tested using `FailOnEnumerateSequence`

Reply:
Adressed in 635af07